.github/workflows/test-e2e-cli.yml

```diff
@@ -11,6 +11,8 @@ env:
   GO_VERSION: 1.22.2
   TEST_PARALLELISM: 8
   TEST_PARALLELISM_PKG: 1
+  # Temporary removed project: https://keboolaglobal.slack.com/archives/C055HSMKX51/p1714024266416349
+  # {"host":"connection.north-europe.azure.keboola.com","project":10892,"stagingStorage":"abs","backend":"snowflake","token":"${{ secrets.TEST_KBC_PROJECT_10892_TOKEN }}", "legacyTransformation": true}
```

Comment:
There is an unrelated issue with the project: https://keboolaglobal.slack.com/archives/C055HSMKX51/p1714024266416349